### PR TITLE
Merge similar funcs

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**

/kind cleanup
/kind design

**What this PR does / why we need it**:

Merge  similar funcs

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/garbagecollector/graph_builder.go [hunk 1]
```diff
@@ -449,19 +449,17 @@ func beingDeleted(accessor metav1.Object) bool {
...
-	}
-	return false
+	return hasFinalizer(accessor, metav1.FinalizerDeleteDependents)
 }
 
 func hasOrphanFinalizer(accessor metav1.Object) bool {
+	return hasFinalizer(accessor, metav1.FinalizerOrphanDependents)
+}
+
+func hasFinalizer(accessor metav1.Object, matchFinalizer string) bool {
```
Comment (yue9944882) [inline pkg/controller/garbagecollector/graph_builder.go]: s/matchFinalizer/matchingFinalizer/

Comment (cwdsuzhou) [reply pkg/controller/garbagecollector/graph_builder.go]: thanks for your kind advice

done

**Comment by cwdsuzhou** [comment]
ping @caesarxuchao for review

**Pull request merged**